jade-repl.el

```diff
@@ -259,7 +265,13 @@ optional."
             (propertize (format "%s:%s" (file-name-nondirectory url) line)
                         'font-lock-face 'jade-link-face
                         'jade-action (lambda ()
-                                       (browse-url url))
+                                       (let ((buffer (get-buffer (file-name-nondirectory url))))
```

Comment:
Would this actually work? What if I have the file on disk but no buffer open? What about multiple files with the same name but in different directories?